{
  "_args": [
    [
      {
        "raw": "hapi-error",
        "scope": null,
        "escapedName": "hapi-error",
        "name": "hapi-error",
        "rawSpec": "",
        "spec": "latest",
        "type": "tag"
      },
      "C:\\Users\\l00251529\\Desktop\\iux_qs"
    ]
  ],
  "_from": "hapi-error@latest",
  "_id": "hapi-error@1.8.0",
  "_inCache": true,
  "_location": "/hapi-error",
  "_nodeVersion": "6.10.0",
  "_npmOperationalInternal": {
    "host": "s3://npm-registry-packages",
    "tmp": "tmp/hapi-error-1.8.0.tgz_1499821170504_0.04234888916835189"
  },
  "_npmUser": {
    "name": "nelsonic",
    "email": "contact.nelsonic@gmail.com"
  },
  "_npmVersion": "3.10.10",
  "_phantomChildren": {},
  "_requested": {
    "raw": "hapi-error",
    "scope": null,
    "escapedName": "hapi-error",
    "name": "hapi-error",
    "rawSpec": "",
    "spec": "latest",
    "type": "tag"
  },
  "_requiredBy": [
    "#USER",
    "/"
  ],
  "_resolved": "http://registry.npmjs.org/hapi-error/-/hapi-error-1.8.0.tgz",
  "_shasum": "f12ec881c71ebd1ae80b9bbcb987b5273ba4d57b",
  "_shrinkwrap": null,
  "_spec": "hapi-error",
  "_where": "C:\\Users\\l00251529\\Desktop\\iux_qs",
  "author": {
    "name": "dwyl & co"
  },
  "bugs": {
    "url": "https://github.com/dwyl/hapi-error/issues"
  },
  "dependencies": {
    "hoek": "^4.1.0"
  },
  "description": "catch errors in your hapi application and display the appropriate error message/page",
  "devDependencies": {
    "boom": "^5.1.0",
    "decache": "^4.1.0",
    "good": "^7.2.0",
    "good-console": "^6.4.0",
    "good-squeeze": "^5.0.2",
    "goodparts": "^1.2.1",
    "handlebars": "^4.0.10",
    "hapi": "^16.4.3",
    "hapi-auth-jwt2": "^7.3.0",
    "istanbul": "^0.4.5",
    "joi": "^10.6.0",
    "jsonwebtoken": "^7.4.1",
    "nodemon": "^1.11.0",
    "pre-commit": "^1.2.2",
    "tap-spec": "^4.1.1",
    "tape": "^4.7.0",
    "vision": "^4.1.0"
  },
  "directories": {},
  "dist": {
    "shasum": "f12ec881c71ebd1ae80b9bbcb987b5273ba4d57b",
    "tarball": "https://registry.npmjs.org/hapi-error/-/hapi-error-1.8.0.tgz"
  },
  "gitHead": "db83dea54344a8e0429ee86f2d24464204c157de",
  "homepage": "https://github.com/dwyl/hapi-error#readme",
  "keywords": [
    "custom",
    "customise",
    "error",
    "friendly",
    "hapi",
    "hapijs",
    "hapi.js",
    "helpful",
    "html",
    "human",
    "json",
    "message",
    "page",
    "useful",
    "user friendly",
    "UX"
  ],
  "license": "GPL-2.0",
  "main": "lib/index.js",
  "maintainers": [
    {
      "name": "nelsonic",
      "email": "contact.nelsonic@gmail.com"
    }
  ],
  "name": "hapi-error",
  "optionalDependencies": {},
  "pre-commit": [
    "check-coverage",
    "lint"
  ],
  "readme": "ERROR: No README data found!",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/dwyl/hapi-error.git"
  },
  "scripts": {
    "check-coverage": "npm run test && node_modules/.bin/istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
    "dev": "PORT=8000 ./node_modules/.bin/nodemon example/server_example.js",
    "lint": "goodparts ./lib",
    "nocov": "./node_modules/tape/bin/tape ./test/*.test.js",
    "start": "node example/server_example.js",
    "test": "istanbul cover ./node_modules/tape/bin/tape ./test/*.test.js  | node_modules/tap-spec/bin/cmd.js"
  },
  "version": "1.8.0"
}
